debian/changelog

```diff
@@ -1,4 +1,63 @@
-ubuntu-advantage-tools (27.1) impish; urgency=medium
+ubuntu-advantage-tools (27.2) impish; urgency=medium
+
+  * d/control:
```

Comment:
I think we need a `postinst` entry for the status patch work

Reply:
Good catch!